.github/workflows/continuous-integration.yml

```diff
@@ -49,26 +48,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 14
-        uses: actions/setup-node@v2
-        with:
-          node-version: 14
+      - uses: actions/setup-node@v2
```

Comment:
Curious what's the benefit of letting GitHub Actions implicitly pick the Node.js version;

My main concern is that a new major Node.js version is going to unexpectedly break the CI workflow; especially since we do not test against Node.js Current versions yet.

Reply:
`setup-node` always uses LTS versions (currently v14). We will always support the LTS version before they change it in GH Actions